src/pages/start-mining/mainnet.md

```diff
@@ -92,43 +92,34 @@ bitcoin-cli -rpcport=8332 -rpcuser=your-user -rpcpassword=your-password importad
 
 Once imported, we need to get some BTC to that address. You should be able to transfer BTC to this address using a crytpocurrency exchange such as [Coinbase](https://www.coinbase.com), [Binance](https://www.binance.com), or [Kraken](https://www.kraken.com).
 
-Now, we need to configure our node to use this Bitcoin keychain. In the `stacks-blockchain` folder, create a new file called `mainnet-miner-conf.toml`.
+Now, we need to configure our node to use this Bitcoin keychain. In the `stacks-blockchain` folder, modify the file at [`testnet/stacks-node/conf/mainnet-miner-conf.toml`](https://github.com/blockstack/stacks-blockchain/blob/master/testnet/stacks-node/conf/mainnet-miner-conf.toml).
 
-Paste in the following configuration:
+Update the following properties:
 
 ```toml
 [node]
-rpc_bind = "0.0.0.0:20443"
-p2p_bind = "0.0.0.0:20444"
+...
 # Enter your private key here
 seed = "replace-with-your-private-key"
-miner = true
+local_peer_seed = "replace-with-your-private-key"
+...
 
 [burnchain]
-chain = "bitcoin"
-mode = "mainnet"
-# peer_host should point to your local bitcoind node
+...
+# To mine on mainnet, you need to run bitcoind locally
+# Details can be found in above section, 'Running bitcoind locally'
 peer_host = "127.0.0.1"
-username = "your-bitcoind-username"
-password = "your-bitcoind-password"
-rpc_port = 8332
-peer_port = 8333
-
-# optional - may help speed up sync time
-[connection_options]
-read_only_call_limit_write_length = 0
-read_only_call_limit_read_length = 100000
-read_only_call_limit_write_count = 0
-read_only_call_limit_read_count = 30
-read_only_call_limit_runtime = 1000000000
+username = "<USERNAME>"
+password = "<PASSWORD>"
+...
 ```
 
 Now, grab your `privateKey` from earlier, when you ran the `make_keychain` command. Replace the `seed` field with your private key. Save and close this configuration file.
 
 To run your miner, run this in the command line:
 
 ```bash
-stacks-node start --config=./mainnet-miner-conf.toml
+stacks-node start --config=./testnet/stacks-node/conf/mainnet-miner-conf.toml
```

Comment:
Is the `testnet` string in this directory listing correct?  I see it's all through the mainnet mining doc so it makes me think it's not a mistake, but is it potentially confusing for someone who is intending to run a miner on the mainnet?

Reply:
Yeah I think the Splat team knows the folder structure needs to be reworked, but right now this is correct. The [source code for the `stacks-node` binary is in this folder](https://github.com/blockstack/stacks-blockchain/tree/master/testnet), as well as the [example config files](https://github.com/blockstack/stacks-blockchain/tree/master/testnet/stacks-node/conf).